packages/form-js-editor/test/spec/properties-panel/groups/GeneralGroup.spec.js

```diff
@@ -367,21 +367,115 @@ describe('GeneralGroup', function() {
     });
 
 
-    it('should render for INPUTS', function() {
+    describe('for singleSelect-like INPUTS', () => {
 
-      // given
-      for (const type of INPUTS.filter(i => ![ 'checklist', 'taglist' ].includes(i))) {
+      const singleSelectInputTypes = ['radio', 'select'];
 
-        const field = { type };
+      it('should NOT render by default', () => {
```

Comment:
What means _default_ here? When `valuesKey` is set? Maybe we should name the test this way then.